release/release_gold.sh

```diff
@@ -270,36 +270,48 @@ function tag_release {
 		repository=liferay-portal
 	fi
 
-	local tag_data=$(
-		cat <<- END
-		{
-			"message": "",
-			"object": "${git_hash}",
-			"tag": "${_PRODUCT_VERSION}",
-			"type": "commit"
-		}
-		END
-	)
-
-	if [ $(invoke_github_api_post "${repository}/git/tags" "${tag_data}") -eq "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
-	then
-		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
-	fi
+	local repository_owners=("liferay-release" "davisaints")
 
-	local ref_data=$(
-		cat <<- END
-		{
-			"message": "",
-			"ref": "refs/tags/${_PRODUCT_VERSION}",
-			"sha": "${git_hash}"
-		}
-		END
-	)
-
-	if [ $(invoke_github_api_post "${repository}/git/refs" "${ref_data}") -eq "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}" ]
-	then
-		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
-	fi
+	for repository_owner in "${repository_owners[@]}"
```

Comment:
@davisaints, no need to declare an array, can you please do the following instead?
```
for repository_owner in brianchandotcom liferay
```